docs/contributing/featureflags.md

```diff
@@ -57,7 +57,8 @@ see immediate results in running / debugging code.
 To set up the local file data store:
 
 1. Create a file named `flags.json` file in the root project directory (e.g. `/src/Api/` for the
-   `Api` project). The name and path can be changed in configuration via `FlagDataFilePath`.
+   `Api` project). The name and path can be changed via the
+   `globalSettings__LaunchDarkly__FlagDataFilePath` environment variable.
```

Comment:
❓ How often is this variable formatting revealed in contributing docs? It implies a runtime environment vs. what's in the application settings themselves and the LHS here is "enough" to me to explain what to change.

Reply:
I think I am good otherwise, but what about this one?

Reply:
Sorry, I overlooked this one.  See if it looks better now.  I also tried to pare down where I was explicitly referring to the file being named `flags.json`, as that is just the default file name.  I referred to it more generically as the `local data store` or `local JSON file`.